qiita_db/analysis.py

```diff
@@ -628,6 +629,40 @@ def remove_samples(self, proc_data=None, samples=None):
 
         conn_handler.executemany(sql, remove)
 
+    def get_changes(self):
+        """If the analysis has altered data, returns the changes list
+
+        Returns
+        -------
+        dict of list
+            Changes in the form {study_title: [change1,change2, ...]...}
+        """
+        if self.status != "altered_data":
+            return {}
+        conn_handler = SQLConnectionHandler()
+        studydict = defaultdict(list)
+        sql = """SELECT DISTINCT study_title, change, timestamp
```

Comment:
timestamp is a reserver word by SQL. Can you change the name of the column so it doesn't use the reserved word?